packages/cli/src/protocols/index.ts

```diff
@@ -290,6 +299,21 @@ const ethereumProtocol: ProtocolConfig = {
   mappingScaffold: EthereumMappingScaffold,
 };
 
+const subgraphProtocol: ProtocolConfig = {
+  displayName: 'Subgraph',
+  abi: EthereumABI,
+  contract: undefined,
+  getTemplateCodeGen: undefined,
+  getTypeGenerator(options) {
+    return new EthereumTypeGenerator(options);
+  },
+  getSubgraph(options) {
+    return new SubgraphDS(options);
+  },
+  manifestScaffold: undefined,
+  mappingScaffold: undefined,
```

Comment:
Any reason we're not using the [subgraph scaffolds](https://github.com/graphprotocol/graph-tooling/tree/b45cf7361075df227c4a3699d884070c02d121b5/packages/cli/src/protocols/subgraph/scaffold) here ?